src/freenas/usr/local/libexec/freenas-debug/active_directory/active_directory.sh

```diff
@@ -49,7 +52,7 @@ active_directory_func()
 	section_footer
 
 	section_header "Active Directory Run Status"
-	service samba_server onestatus
+	service winbindd status
```

Comment:
Will this work in scale ?

Reply:
I was planning to alias the systemd config on Linux so that we're consistent in this between FreeBSD / Linux.